test/games/strategy/triplea/ui/TestRoute.java

```diff
@@ -0,0 +1,64 @@
+package games.strategy.triplea.ui;
```

Comment:
Separation of concerns. Isolate the code that does the drawing, test that manually. Basically break things up into classes, not just methods. A class can be mocked and tested independently. Eventually then the parts do the actual UI work, that interacts with the Graphics object, will be just about as small as possible, and it is is okay to test that manually.